### DISCUSSION TITLE
Daily 06/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#44] [Daily]
Con respecto al proyecto:

¿Qué hice ayer?
¿Qué hice o haré hoy?
Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer terminé de crear las issue de las US. También estuve agregando requerimientos funcionales y no funcionales en el google doc y  el enlace dentre de la wiki y otros detalle que faltaban en la documentación. Por la tarde participé en la clase del instituto. Por otro lado estuve viendo material para hacer los digramas de clase de de casos de uso.
Hoy voy a continuar con los diagramas y con las tareas que tengo pendientes en la plataforma. Por la noche tengo clase de inglés.

**Comment by BraianTroncoso** [comment]
Ayer terminé las clases que tenia pendientes. Probé el esqueleto que compartió el profe sobre Android y funcionaba perfecto. Hoy tengo clases de Java con sprint-boot, también node.
